go/api/base_client.go

```diff
@@ -540,3 +566,50 @@ func (client *baseClient) PingWithMessage(message string) (string, error) {
 	}
 	return response.Value(), nil
 }
+
+func (client *baseClient) ConfigGet(args []string) (map[Result[string]]Result[string], error) {
+	res, err := client.executeCommand(C.ConfigGet, args)
+	if err != nil {
+		return nil, err
+	}
+
+	return handleStringToStringMapResponse(res)
+}
+
+func (client *baseClient) ConfigSet(parameters map[string]string) (Result[string], error) {
+	result, err := client.executeCommand(C.ConfigSet, utils.MapToString(parameters))
+	if err != nil {
+		return CreateNilStringResult(), err
+	}
+
+	return handleStringResponse(result)
+}
+
+func (client *baseClient) CustomCommand(args []string) (interface{}, error) {
+	res, err := client.executeCommand(C.CustomCommand, args)
+	if err != nil {
+		return nil, err
+	}
+
+	resString, err := handleStringOrNullResponse(res)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
These should be in respective `glide_client` and `glide_cluster_client` files as their behavior is not common.